build.gradle.kts

```diff
@@ -28,6 +28,9 @@ java {
     }
 }
 
+tasks.withType<JavaCompile> {
+    options.release.set(8)
```

Comment:
This would override the default to set the target to JDK 8 ?

Reply:
Yes